code/game/objects/items/devices/guitar.dm

```diff
@@ -0,0 +1,41 @@
+//copy pasta of the space piano, don't hurt me -Pete
+
+/obj/item/device/guitar
+	name = "guitar"
+	desc = "It's made of wood and has bronze strings."
+	icon = 'icons/obj/musician.dmi'
+	icon_state = "guitar"
+	item_state = "guitar"
+	icon_override = 'icons/mob/in-hand/tools.dmi'
+	force = 10
+	var/datum/song/handheld/song
+
+/obj/item/device/guitar/New()
+	song = new("guitar", src)
+	song.instrumentExt = "ogg"
+
+/obj/item/device/guitar/Destroy()
+	del(song)
```

Comment:
it can't, sadly--TG's garbage collector can; ours just throws a "HEY ONLY ATOM MOVEABLE!"